pkg/plugins/plugins.go

```diff
@@ -164,11 +164,14 @@ func scan(pluginDir string) error {
 }
 
 func (scanner *PluginScanner) walker(currentPath string, f os.FileInfo, err error) error {
+	// TODO: Based on how we search for plugin.json in grafan-net fetchPluginJson, it should be enough to scan
+	// just /src, /dist or / paths inside the plugin dir
```

Comment:
Not sure if having plugin.json anywhere in the package is something we want to support, it is not something we support for packages registered in grafana.com API

Reply:
Based on https://grafana.com/docs/plugins/developing/code-styleguide/#plugin-json-mandatory i would say only check `/src` or `/dist`.

Reply:
The problem is dealing with plugins that embed other plugins, like https://github.com/raintank/worldping-app/tree/master/dist/grafana-worldmap-panel

Within an individual plugin we expect plugin.json to be in those locations, but we walk all the way down through the plugin to find any embedded plugins.